### ISSUE TITLE
Citation information by pinging (say add in config or ?cite call)

### ISSUE THREAD

**Issue by MansMeg** [statisticssweden/PxWeb#4]
A lot of people use the data and want to cite it correctly. By just adding citation information we can handle everything in the pxweb R package. This is good also for data providers to be cited correctly.

Best is if you (together with say Jan at Stat Norway) set up a correct BiBTeX reference structure to use for a given table. Then we could adapt to that one and just pass it along.

**Comment by janbrus** [comment]
We discussed this request at Statstics Norway this a year ago. We wanted not use our development resources to implement this for one kind of reference manager and not others. https://en.wikipedia.org/wiki/Comparison_of_reference_management_software

**Comment by MansMeg** [comment]
Of course, any citation structure would work. Best would be to ping and then get a reference format of choice. Say ?ref=bibtex as url arg or similar. The purpuse is just to get _any_ way to create a correct reference for a given table.
